smart-contracts/contracts/lp-strategy/src/proptests.rs

```diff
@@ -271,7 +261,7 @@ mod tests {
                             amount,
                             &Coin {
                                 denom: "ibc/local_osmo".to_string(),
-                                amount: failed_total_amount + pending_total_amount,
```

Comment:
We should also verify here that the failed bonds are treated properly

Reply:
done